### DISCUSSION TITLE
LHC 4.26 and PHP 5.6

### DISCUSSION THREAD

**Discussion by Rembacher** [LiveHelperChat/livehelperchat#1989] [Q&A] [answered]
Hi Everyone, I read in this https://forum.livehelperchat.com/viewtopic.php?id=3890 that the latest LHC should run on PHP 5.6.

I installed LHC 4.26 via cpanel and Softaculous(as I'm not skilled to manually install apps) successfully changing my php version to 7.4 in cpanel. Of course when I reversed to php 5.6 the URL https://www.mysite.com/helper2023/index.php/site_admin/user/login didn't work.

Would just like to confirm that LHC 4.26 isn't compatible with php 5.6.

Thank you very much,
Richard

**Comment by remdex** [comment, marked as answer]
Yes, please use new php version...

**Reply by Rembacher** [reply]
Thanks for the confirmation
